frontend/projects/abtesting/src/app/features/dashboard/experiment-users/validator/experiment-users-validators.ts

```diff
@@ -5,11 +5,6 @@ export class ExperimentUserValidators {
   static validateExcludedEntityForm(controls: AbstractControl): { [key: string]: any } | null {
     const entityValue = controls.get('entityType').value;
     const groupValue = controls.get('groupType').value;
-    const customGroupValue = controls.get('customGroupName').value;
-    const groupTypeOther = 'other';
-    if (groupValue === groupTypeOther && entityValue === EntityTypes.GROUP_ID) {
-      return !!customGroupValue ? null : { customGroupNameError: true };
-    }
     return null;
```

Comment:
What's the purpose of returning null directly? If this function is not useful, remove it.

Reply:
code updated